code/src/App.js

```diff
@@ -1,9 +1,81 @@
-import React from 'react';
+/* eslint-disable no-underscore-dangle */
+import React, { useEffect, useState } from 'react'
+import ThoughtForm from 'components/ThoughtForm'
+import ThoughtItem from 'components/ThoughtItem'
+import Spinner from 'components/Spinner'
+
+const FETCH_API = 'https://happy-thoughts-ux7hkzgmwa-uc.a.run.app/thoughts'
 
 export const App = () => {
+  const [thoughts, setThoughts] = useState([])
+  const [loading, setLoading] = useState(false);
+  const [newThought, setNewThought] = useState('')
+
+  const fetchThoughts = () => {
+    setLoading(true)
+    fetch(FETCH_API)
+      .then((res) => res.json())
+      .then((data) => setThoughts(data))
+      .finally(() => setLoading(false));
+  }
+
+  useEffect(() => {
+    fetchThoughts()
+  }, [])
+
+  const handleFormCleanup = () => {
```

Comment:
I recall this being mentioned but did not find use for it myslef. I take it that it is good practice but I fail to see it's function for this project...? Might just be that our code differ that much. :)